readthedocs.yml

```diff
@@ -1,14 +1,13 @@
 version: 2
 
+mkdocs:
```

Comment:
If you build in commands using `python -m mkdocs build --clean --site-dir $READTHEDOCS_OUTPUT/html --config-file mkdocs.yml`, is this section still necessary?

Reply:
just tested--not necessary